### PR TITLE
Wording Change: automatic runtime to compute session

### PR DESCRIPTION
**Pull request by crazygao**
# Description

automatic runtime to compute session

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow-azure/promptflow/azure/operations/_run_operations.py [hunk 1]
```diff
@@ -643,7 +643,7 @@ def stream(self, run: Union[str, Run], raise_on_error: bool = True, timeout: int
                         f"The run {run.name!r} is in status {run.status} and produce no new logs for {timeout} seconds,"
                         "streaming is stopped. If the final status is 'NotStarted', "
                         "Please make sure you are using the latest runtime.\n"
-                        "For automatic runtime case, please try extending the timeout value.\n"
```
Comment (D-W-) [inline src/promptflow-azure/promptflow/azure/operations/_run_operations.py:646]: Chang to use constant?

Comment (crazygao) [reply src/promptflow-azure/promptflow/azure/operations/_run_operations.py:646]: OK

File: src/promptflow-azure/promptflow/azure/operations/_run_operations.py [hunk 2]
```diff
@@ -643,7 +643,7 @@ def stream(self, run: Union[str, Run], raise_on_error: bool = True, timeout: int
                         f"The run {run.name!r} is in status {run.status} and produce no new logs for {timeout} seconds,"
                         "streaming is stopped. If the final status is 'NotStarted', "
                         "Please make sure you are using the latest runtime.\n"
```
Comment (D-W-) [inline src/promptflow-azure/promptflow/azure/operations/_run_operations.py:645]: runtime -> session?

Comment (crazygao) [reply src/promptflow-azure/promptflow/azure/operations/_run_operations.py:645]: Yes.

File: docs/cloud/azureai/create-run-with-compute-session.md [hunk 3]
```diff
@@ -1,16 +1,15 @@
-# Create run with automatic runtime
+# Create run with compute session
```
Comment (wangchao1230) [inline docs/cloud/azureai/create-run-with-compute-session.md:1]: Please check the work item, we also need to do some reorg of the pages. Let's sync tommorow

Comment (crazygao) [reply docs/cloud/azureai/create-run-with-compute-session.md:1]: OK

**Review by wangchao1230 — APPROVED** [review]

**Review by D-W- — APPROVED** [review]

**Pull request merged**